src/patternfly/components/AppLauncher/app-launcher-menu-search.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-app-launcher__menu-search{{#if app-launcher-menu-search--modifier}} {{app-launcher-menu-search--modifier}}{{/if}}"
```

Comment:
In the context selector, this wrapper is named as `pf-c-context-selector__menu-input`. I think we should be consistent here. I don't have an opinion on the name I think search is definitely more specific if know its only going to be search there. We could update the context selector in a breaking change too.

Reply:
Agreed, I'd like to update the context selector in a breaking change release to rename it. I went ahead and renamed it here to avoid a breaking change in the future, or do you think we should name this `menu-input` now and update them both in a breaking change release?

Reply:
I think search is good here and then create a follow up issue to change the context selector

Reply:
https://github.com/patternfly/patternfly-next/issues/2438